src/applications/personalization/profile360/util/index.js

```diff
@@ -59,26 +62,33 @@ export const hasInvalidWorkPhoneNumberError = errors =>
   hasErrorMessage(errors, GENERIC_ERROR_KEY, 'day phone number') ||
   hasErrorMessage(errors, GENERIC_ERROR_KEY, 'day area number');
 
+export const hasPaymentRestrictionIndicatorsError = errors =>
+  hasErrorMessage(errors, PAYMENT_RESTRICTIONS_PRESENT_KEY);
+
 // Helper that creates and returns an object to pass to the recordEvent()
 // function when an errors occurs while trying to save/update a user's direct
 // deposit payment information. The value of the `error-key` prop will change
 // depending on the content of the `errors` array.
-export const createDirectDepositAnalyticsDataObject = errors => {
+export const createDirectDepositAnalyticsDataObject = (errors = []) => {
   const key = 'error-key';
-  const eventDataObject = {
+  let errorCode = GA_ERROR_KEY_DEFAULT;
+  if (hasFlaggedForFraudError(errors)) {
```

Comment:
This list of conditionals is gross and I'd like to follow up on later and fix. But didn't want to blow up the scope of this particular PR.

Reply:
Yeah, my first instinct is that each DD error code could be mapped to the GA event like ->

```js
const GA_EVENTS = {
    [INVALID_ROUTING_NUMBER_KEY]: 'invalid-routing-number-error',
    // ...
}
```

But that's not really useful, because when `createDirectDepositAnalyticsDataObject` in the action creator we don't yet know what the DD error is. The  refactor would probably involve moving the error messaging into Redux state, so that the analytics event and the DD error message are next to each other